MSBuild.Sdk.Extras/build/platforms/MonoAndroid.targets

```diff
@@ -11,6 +11,7 @@
 		<AndroidResgenFile Condition="'$(AndroidResgenFile)' == ''">$(IntermediateOutputPath)$(TargetFramework)\Resource.Designer$(_SdkLanguageExtension)</AndroidResgenFile>
 		<!-- This is here to prevent a warning in the Xamarin.Android.Common.Debugging.targets when a blank is passed into _GetPrimaryCpuAbi -->
 		<AdbTarget Condition="'$(AdbTarget)' == ''">none</AdbTarget>
+		<AndroidClassParser Condition="'$(AndroidClassParser)' == ''">class-parse</AndroidClassParser>
```

Comment:
I'd move this to a separate PR since it's unrelated to binding projects, right?

Reply:
If this won't be set, `jar2xml` will be used as default, but that is the old way of converting Android projects.